byline.go

```diff
@@ -145,17 +145,15 @@ func (lr *Reader) MapStringErr(filterFn func(string) (string, error)) *Reader {
 	})
 }
 
-// Each - processing each line without changing the line
+// Each - processing each line
```

Comment:
Yes, slice can be overwritten in next filters. Small example:
```Go
	reader := strings.NewReader("111\n222\n333")

	i := 0
	var firstLine []byte
	out, _ := byline.NewReader(reader).Each(func(line []byte) {
		i++
		if i == 1 {
			firstLine = line
		}
	}).Each(func(line []byte) {
		line[0] = 'Z'
	}).ReadAllString()

	fmt.Println(string(out))
	fmt.Println("firstLine:", string(firstLine))
```
Added comment about this in d8e92fd28adcac0209fb477b47401093d4f853be